modules/interfaces/package.json

```diff
@@ -20,6 +20,7 @@
     "watch": "upbin tsc --declaration --watch"
   },
   "dependencies": {
+    "@types/node": "^12.0.2",
```

Comment:
move `@types` import to root `devDependencies`

Reply:
removed because it becomes unused.